.github/workflows/test.yml

```
@@ -0,0 +1,107 @@
+name: Tests
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)